docker/ubuntu-based/prod/vswitch/Dockerfile

```diff
@@ -1,15 +1,15 @@
 FROM ubuntu:16.04
 
 RUN apt-get update \
- && apt-get install -y \
+    && apt-get install -y \
```

Comment:
@nbartos: Can you leave these as space-&&, please? This is how I've formatted some other Dockerfiles. This makes them easier to read when all the && line up under the RUN commands.